[2024-09-22T10:49:17.881+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flats.transform_to_dims manual__2024-09-22T10:48:40.357733+00:00 [queued]>
[2024-09-22T10:49:17.902+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flats.transform_to_dims manual__2024-09-22T10:48:40.357733+00:00 [queued]>
[2024-09-22T10:49:17.908+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-22T10:49:17.988+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transform_to_dims> on 2024-09-22 10:48:40.357733+00:00
[2024-09-22T10:49:18.007+0000] {standard_task_runner.py:57} INFO - Started process 2242 to run task
[2024-09-22T10:49:18.022+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'flats', 'transform_to_dims', 'manual__2024-09-22T10:48:40.357733+00:00', '--job-id', '1112', '--raw', '--subdir', 'DAGS_FOLDER/flats.py', '--cfg-path', '/tmp/tmp5lf8it__']
[2024-09-22T10:49:18.030+0000] {standard_task_runner.py:85} INFO - Job 1112: Subtask transform_to_dims
[2024-09-22T10:49:18.105+0000] {task_command.py:415} INFO - Running <TaskInstance: flats.transform_to_dims manual__2024-09-22T10:48:40.357733+00:00 [running]> on host 1646e3dc8472
[2024-09-22T10:49:18.208+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flats' AIRFLOW_CTX_TASK_ID='transform_to_dims' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T10:48:40.357733+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T10:48:40.357733+00:00'
[2024-09-22T10:49:18.251+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/etl.py:186 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-09-22T10:49:18.253+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-09-22T10:49:18.268+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=flats, task_id=transform_to_dims, execution_date=20240922T104840, start_date=20240922T104917, end_date=20240922T104918
[2024-09-22T10:49:18.329+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-22T10:49:18.380+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
